internal/evaluation_target/application_snapshot_image/application_snapshot_image.go

```diff
@@ -252,41 +252,41 @@ func (a ApplicationSnapshotImage) ValidateAttestationSyntax(ctx context.Context)
 		return errors.New("no attestation data")
 	}
 
-	allErrors := map[string][]jsonschema.KeyError{}
+	var validationErr *jsonschema.ValidationError
 	for _, sp := range a.attestations {
 		pt := sp.PredicateType()
 		if schema, ok := attestationSchemas[pt]; ok {
 			// Found a validator for this predicate type so let's use it
 			log.Debugf("Attempting to validate an attestation with predicateType %s", pt)
-			if errs, err := schema.ValidateBytes(ctx, sp.Statement()); err != nil {
-				// Error while trying to validate
+
+			var st any
+			if err := json.Unmarshal(sp.Statement(), &st); err != nil {
 				return fmt.Errorf("unable to decode attestation data from attestation image: %w", err)
-			} else {
-				if len(errs) == 0 {
-					log.Debugf("Statement schema was validated successfully against the %s schema", pt)
-				} else {
-					log.Debugf("Validated the statement against %s schema and found the following errors: %v", pt, errs)
-					allErrors[pt] = errs
+			}
+
+			if err := schema.Validate(st); err != nil {
+				if validationErr, ok = err.(*jsonschema.ValidationError); !ok {
```

Comment:
If there are two attestations, wouldn't the schema validation errors on the first one be thrown out if the second one validates successfully?

Reply:
Now collects all `jsonschema.ValidationError` errors, similar to how it was done prior